index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Home</title>
+  <link href="./styles/normalize.css" rel="stylesheet">
+  <link href="./styles/icono.min.css" rel="stylesheet" >
+  <link href="./styles/main.css" rel="stylesheet">
+  <script src="https://use.fontawesome.com/ed5b29bc68.js"></script>
+</head>
+
+<body>
+  <header>
+    <li><a href="index.html"><i class="icono-home"></i></a></li>
+    <nav>
+      <li><a href="portfolio.html"> My Pieces </a></li>
+      <li><a href="code-journal.html"> Learning </a></li>
+      <li><a href="about.html"> Resume </a></li>
+    </nav>
+  </header>
+  <section id="index-section">
+    <div>
+      <h2> My Journey to Become a Software Developer </h2>
+    </div>
+    <div class="name">
+      <ul>
+        <li class="star">★</li>
+        <li><h1 id="name-h1"> Sofia Kim </h1></li>
```

Comment:
Great job running this through the HTML validator! The next step would be to approach this with accessibility in mind. (Are the colors, fonts and text size easy to read? Are elements appropriately spaced? Is it clear that the links are clickable? If certain elements of your page did not load, would someone still be able to navigate the page?) I think you may have a lecture on this topic later but I found that these questions helped in driving the direction of a project's design. Great job!